fuse_constraints/include/fuse_constraints/normal_delta_pose_2d_cost_functor.h

```diff
@@ -72,6 +73,8 @@ namespace fuse_constraints
 class NormalDeltaPose2DCostFunctor
 {
 public:
+  FUSE_MAKE_ALIGNED_OPERATOR_NEW();
```

Comment:
Remove, you don't need it for https://github.com/locusrobotics/fuse/pull/65/files#diff-82ac2fce42490aae65c9dfd69e4fd5eeR104 (see https://eigen.tuxfamily.org/dox/group__TopicFixedSizeVectorizable.html)

Reply:
It looks like the `Eigen::MatrixXd A_` attribute could have been of `Eigen::Matrix3d` type, in which case you do need this. But from the comment in the constructor the number of columns is variable.

Reply:
The cost functor supports partial measurements, so the number of output residuals may not be 3. And thus `A_` will be a Nx3 matrix.
That said, there may be a less confusing way to implement this. I'm making a note to review these partial measurement constraints in the future.